web/utils/uris.ts

```diff
@@ -0,0 +1,3 @@
+export default function isValidURI(uri: string): boolean {
+  return uri.includes(':');
```

Comment:
Maybe we can also check for a top level domain. Right now if somebody types `https:` it'll be seen as a completely valid input.